plugins/linear-genome-view/src/LinearGenomeView/components/LinearGenomeView.tsx

```diff
@@ -44,7 +46,8 @@ const useStyles = makeStyles()(theme => ({
   },
 }))
 
-const LinearGenomeView = observer(({ model }: { model: LGV }) => {
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const LinearGenomeView = observer(({ model }: { model: any }) => {
```

Comment:
Any reason why this is no longer an LGV type?

Reply:
MultilevelLinearGenomeView "extends" linear genome view and its model gets passed in here -- it's technically not LGV and if this isn't any it can't reuse the components it needs to

Reply:
I tried restoring type LGV here on my local machine and I think it does not cause a typescript error as far as I can tell. the MultiLevelLinearGenomeView may be "similar enough' that it does not care